### PR TITLE
add 0.1.6 release note

### PR DESCRIPTION
**Pull request by freeznet**

### CODE REVIEW

**Comment by freeznet** [comment]
@Huanli-Meng PTAL when you have time, thanks.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Function Mesh v0.1.6 Release Notes
+
+## v0.1.6 What's New
+
+- Move function runner image to java11 to support Pulsar 2.8 ([#202](https://github.com/streamnative/function-mesh/pull/202))
+- Fixed max replicas value in mesh worker service ([#212](https://github.com/streamnative/function-mesh/pull/212))
+- Update sink/source component observation logic ([#217](https://github.com/streamnative/function-mesh/pull/217))
+- Add function/sink/source status API support with mesh worker service ([#220](https://github.com/streamnative/function-mesh/pull/220)) ([#224](https://github.com/streamnative/function-mesh/pull/224)) ([#225](https://github.com/streamnative/function-mesh/pull/225))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:8]:
```suggestion
- Support Pulsar Functions and connectors (sinks and sources) status API in Function Mesh Worker service. ([#220](https://github.com/streamnative/function-mesh/pull/220)) ([#224](https://github.com/streamnative/function-mesh/pull/224)) ([#225](https://github.com/streamnative/function-mesh/pull/225))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+- Allow set service account name for running pod ([#226](https://github.com/streamnative/function-mesh/pull/226))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:9]:
```suggestion
- Support setting the service account name for running Pods. ([#226](https://github.com/streamnative/function-mesh/pull/226))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+- Fixed NPE when not set AutoAck ([#234](https://github.com/streamnative/function-mesh/pull/234))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:10]:
```suggestion
- Fix the NPE that is triggered when the `AutoAck` option is not set. ([#234](https://github.com/streamnative/function-mesh/pull/234))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+- Fixed mesh worker service get status with fault pod ([#239](https://github.com/streamnative/function-mesh/pull/239))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:11]:
```suggestion
- Fix the issue that the Function Mesh worker service gets status with a fault pod. ([#239](https://github.com/streamnative/function-mesh/pull/239))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,18 @@
...
+- Remove logTopic from source/sink CRD ([#242](https://github.com/streamnative/function-mesh/pull/242))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:12]:
```suggestion
- Remove the `logTopic` option from source  and sink CRDs. ([#242](https://github.com/streamnative/function-mesh/pull/242))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,18 @@
...
+- Fixed service name observe ([#230](https://github.com/streamnative/function-mesh/pull/230))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:13]:
```suggestion
- Fix the observation logic of the service name. ([#230](https://github.com/streamnative/function-mesh/pull/230))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,18 @@
...
+- Bump Pulsar to 2.8 ([#240](https://github.com/streamnative/function-mesh/pull/240))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:14]:
```suggestion
- Bump Pulsar to version 2.8.0. ([#240](https://github.com/streamnative/function-mesh/pull/240))
```

**Review by Huanli-Meng — APPROVED** [review]
Leave my comments. PTAL. BTW, if you agree with my comments, you can just click the **Commit suggestion** button.

**Review by nlu90 — APPROVED** [review]

**Pull request merged**